lesson_07/dags/enrich_user_profiles_pipeline_dag.py

```diff
@@ -0,0 +1,47 @@
+"""
+Enrich_user_profiles processing pipeline
+"""
+import datetime as dt
+
+from airflow import DAG
+from airflow.providers.google.cloud.operators.bigquery import BigQueryInsertJobOperator
+
+
+DEFAULT_ARGS = {
+    'depends_on_past': True,
+    'email_on_failure': True,
+    'email_on_retry': False,
+    'retries': 0,
+    'retry_delay': 5,
+}
+
+dag = DAG(
+    dag_id="enrich_user_profiles_pipeline_dag",
+    description="Ingest and process sales data",
+    schedule_interval=None,
+    start_date=dt.datetime(2022, 9, 1),
+    catchup=True,
+    tags=['enrich_user_profiles'],
+    default_args=DEFAULT_ARGS,
+)
+
+dag.doc_md = __doc__
+
+
+transfer_from_dwh_silver_to_dwh_gold = BigQueryInsertJobOperator(
```

Comment:
How do user profiles come to Silver?
Ther're should be a separate pipeline for user profiles transferring. Nobody knows how this data will be used in future.